src/d3/hapticplot.d3.ts

```diff
@@ -46,31 +50,43 @@ export class Hapticplot{
       .attr('radius', size)
       // Enables controller interaction with points using superhands' tags
       .attr('hoverable', '')
-      .attr('grabbable', '')
-      .attr('stretchable', '')
-      .attr('draggable', '')
-      .attr('dropppable', '')
       // Adds listeners for state change events, which trigger a change in the
       // point's color property when a hover event occurs
-      .on('stateadded',  (d, i, g) => this.modifyColorOnStateChange(g[i], 'hovered', 'orange'))
-      .on('stateremoved',  (d, i, g) => this.modifyColorOnStateChange(g[i], 'hovered', 'green'));
+      .on('hover-start',  (d, i, g) => this.onHoverStart(g[i], d, 'red'))
+      .on('hover-end',  (d, i, g) => this.onHoverEnd(g[i], 'green'));
 
   }
 
-  private modifyColorOnStateChange(entity, state , color){
-    if (d3.event.detail === state){
-      d3.select(entity).attr('color', color);
-    }
-  }
-
-  // Generates a world space position for each data entity, based on ingested data
   private generatePositions(data, index){
+    /*
+      Generates a world space position for each data entity, based on ingested data
+    */
```

Comment:
I think this should be a JSDoc comment (use two asterixes and put it outside the method). Same elsewhere in this file.

Reply:
I switched them over!